sympy/core/function.py

```diff
@@ -1270,19 +1270,20 @@ def __new__(cls, expr, *variables, **kwargs):
                         must be supplied to differentiate %s''' % expr))
 
         # Standardize the variables by sympifying them:
-        variables = list(sympify(variables))
+        #variables = sympify(list(variables))
```

Comment:
This is actually not needed here since all variables are eventually stored in `Tuple` (or `Array` which again stores it in `Tuple`) where `sympify` is also used.

Reply:
Maybe this should just be deleted.